rest-api-spec/pom.xml

```diff
@@ -0,0 +1,37 @@
+<project>
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.elasticsearch</groupId>
+  <artifactId>rest-api-spec</artifactId>
+  <version>2.0.0-SNAPSHOT</version>
+  <name>Elasticsearch Rest API Spec</name>
+  <parent>
+      <groupId>org.sonatype.oss</groupId>
+      <artifactId>oss-parent</artifactId>
```

Comment:
Can't we inherit from parent pom here?

So we control maven-remote-resources-plugin version from there?